scania-digital-design-system/tegel
# test(banner): Adding unit tests

**Describe pull-request**  
Adds units tests for banner component

**Solving issue**  
Fixes: [CDEP-3005](https://tegel.atlassian.net/browse/CDEP-3005)

**How to test**  
1. Checkout code
2. Make sure `npm run test` completes and that it runs without errors

[CDEP-3005]: https://tegel.atlassian.net/browse/CDEP-3005?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
